modules/alloy/CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- Added new `retargetAndDispatchWith` API to `AlloyTriggers` to allow for redispatching of events to different targets. #TINY-8952
+- Added new `refetch` API to `Dropdown` that triggers a fetch and redraw for the dropdown menu. #TINY-8952
+- Added new `onHighlightItem` and `onDehighlightItem` settings to `TieredMenu`, and deprecated the existing `onHighlight`. #TINY-8952
+- Added new `getExistingCoupled` API to `Coupling` that gets an existing coupled component, but does not create it if it doesn't already exist. #TINY-8952
+
+
+### Changed
+- The `highlightImmediately` setting for `TieredMenu` was replaced with the more granular `HighlightOnOpen`. #TINY-8952
+
+### Fixed
```

Comment:
You've got 2 fixed sections in this change log now. The order isn't quite right either, it should be `Added` -> `Improved` -> `Changed` -> `Deprecated` -> `Fixed` -> `Removed`.

Reply:
- [ ] fix changelog order

Reply:
Updated.